installer/roles/kubernetes/tasks/main.yml

```diff
@@ -246,6 +246,8 @@
 
 - name: Migrate database
   shell: |
+    {{ kubectl_or_oc }} -n {{ kubernetes_namespace }} exec ansible-tower-management -- \
+      bash -c "echo 'DJANGO_REDIS_IGNORE_EXCEPTIONS = True' > /etc/tower/conf.d/redis.py"
```

Comment:
@shanemcd @rebeccahhh What's the reasoning for not *always* doing this?  It seems like the old python-memcached library always operated with silent cache misses.

Reply:
I just worry that it will break sometime and we'll have no visibility into it until a user or customer finds it.